vars/main.yml

```diff
@@ -1,3 +1,4 @@
 # vars file for docker-machine
 ---
 docker_machine_downloads_path: /var/lib/ansible/docker-machine/downloads
+docker_machine_mirror: https://github.com
```

Comment:
Can you rename it to `docker_machine_download_url` and move it to `defaults/main.yml`?

Reply:
Hey @tersmitten, sure updated!